setup.examples/exim/etc/exim4/acl_check_recipient_relay

```diff
@@ -0,0 +1,12 @@
+# Config for MailScanner Gateway
```

Comment:
Changed the filename from `acl_check_recipient_relay` to `mailscanner_acldefs`

Updated content.

```
# Config for MailScanner Gateway
# /etc/exim4/mailscanner_acldefs
# Use with CHECK_RCPT_LOCAL_ACL_FILE = CONFDIR/mailscanner_acldefs

# To activate sender callback on remote host (mailhub) at SMTP time
.ifdef RECIPIENT_VERIFY_CALLBACK_RELAY
# Recipient verification for relay domains with callback
deny    !acl           = acl_local_deny_exceptions
        log_message    = 550 Unknown user ${local_part}@${domain} (Recipient relay callback verification failed)
        message        = 550 Unknown user
      ! verify         = recipient/defer_ok/callout=10s,defer_ok
.endif

```